api/v1alpha8/openstackcluster_webhook.go

```diff
@@ -63,6 +63,23 @@ func (r *OpenStackCluster) ValidateCreate() (admission.Warnings, error) {
 		allErrs = append(allErrs, field.Forbidden(field.NewPath("spec", "identityRef", "kind"), "must be a Secret"))
 	}
 
+	if r.Spec.ManagedSecurityGroups.Enabled && len(r.Spec.ManagedSecurityGroups.AllNodesSecurityGroupRules) > 0 {
+		for _, rule := range r.Spec.ManagedSecurityGroups.AllNodesSecurityGroupRules {
+			if rule.Name == "" {
+				allErrs = append(allErrs, field.Required(field.NewPath("spec", "managedSecurityGroups", "allNodesSecurityGroupRules", "name"), "name is required"))
+			}
```

Comment:
Just to double check, is that comment just for "name" validation? Or also for other fields. I think we need to validate other fields, since they aren't required.

Reply:
This comment was specific to `name`.